pyblish_qml/control.py

```diff
@@ -1016,7 +1018,7 @@ def run(self, plugins, context, callback=None, callback_args=[]):
         # the GUI and commence next task.
         def on_next(result):
             if isinstance(result, StopIteration):
-                return on_finished(str(result))
+                return on_finished()
 
```

Comment:
I found that if we keep this line unchanged, and instead of changing `raise StopIteration` to `return log.error`, just `return StopIteration` will fix for Py37 and keep the message showing properly on GUI !

Reply:
Oops, just found out that this has been merged :P
I'll submit another PR then :)

Reply:
Great news 👍